### DISCUSSION TITLE
Help to Achieve Galaxy Brain Acheivement

### DISCUSSION THREAD

**Discussion by DevPro15** [Schweinepriester/github-profile-achievements#441] [Q&A]
Best Way to increase your achievements on githhub?

**Comment by heyanik** [comment]
Okay

**Comment by montasim** [comment]
@Zair15 in this way will not be able to get the Galaxy Brain badge. Your answer will be needed to be selected two times. Not to mention you can not select your answer. Your answer needs to be selected by another user. Try commenting on another discussion and ask them to select your answer. Best of luck.

By the way, can you mark this as an answer?

**Comment by 1RB** [comment]
comment on other discussions and ask them to mark yours as the answer

**Comment by abidurrahman11** [comment]
Guys try to help others with issues. If you answer them and if they mark it, then you'll get the badge.
Don't ask someone for a badge. You can always get reward if you truly help others.
Best of luck😍🤩
